CMakeLists.txt

```diff
@@ -305,16 +297,16 @@ if(NOT WIN32)
     list(APPEND BFSYSROOT_CMAKE_ARGS
         -DBAREFLANK_SOURCE_DIR=${CMAKE_SOURCE_DIR}
         -DBAREFLANK_BINARY_DIR=${CMAKE_BINARY_DIR}
-        -DCMAKE_INSTALL_PREFIX=${BAREFLANK_INSTALL_PREFIX}
+        -DCMAKE_INSTALL_PREFIX=${BF_BUILD_INSTALL_DIR}
         -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
         -DCMAKE_TOOLCHAIN_FILE=${VMM_TOOLCHAIN_FILE}
         -DENABLE_COVERAGE=${ENABLE_COVERAGE}
         -DENABLE_DYNAMIC_ASAN=${ENABLE_DYNAMIC_ASAN}
         -DENABLE_DYNAMIC_USAN=${ENABLE_DYNAMIC_USAN}
         -DENABLE_TIDY=${ENABLE_TIDY}
         -DENABLE_ASTYLE=${ENABLE_ASTYLE}
-        -DBUILD_SHARED_LIBS=${BAREFLANK_HOST_SHARED_LIBS}
-        -DBUILD_STATIC_LIBS=${BAREFLANK_HOST_STATIC_LIBS}
+        -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
```

Comment:
No way this is going to work. We have "HOST" and "TEST" static / shared variables because unit tests always need to be compiled using static libraries, while the cross compiled components can be static or shared depending on what the user is looking for.

Reply:
The variable BAREFLANK_HOST_STATIC_LIBS is always set to OFF under all build configurations, which makes it unnecessary in my opinion. You can decide if you need to build shared vs. static libs using only the variables BAREFLANK_HOST_SHARED_LIBS (now called BUILD_SHARED_LIBS) and the variable ENABLE_UNITTESTING. In future PRs, I would like to remove these global BUILD_SHARED and BUILD_STATIC variables in favor of providing configurable shared vs. static options for each individual project that produces a library,     with the option of building both if desired. You can do this with cmake by using a second call to add_library and configuring it for shared vs. static.

Reply:
Thats fine, so long as static and shared builds are being compiled as expected. I’m all for simplifying this, I just want to make sure all of the functionality is still there. In general, by default, the VMM and sysroot only compile static libraries and link to a single executable. If shared is stated, you would end up with shared libraries for the VMM and sysroot, but unit tests are still always compiled as static (unit tests always have to be static).